server/src/main/kotlin/suwayomi/tachidesk/manga/impl/Chapter.kt

```diff
@@ -119,54 +120,66 @@ object Chapter {
 
         val numberOfCurrentChapters = getCountOfMangaChapters(mangaId)
         val chapterList = source.getChapterList(sManga)
+        val now = Instant.now().epochSecond
+        val chaptersInDb =
+            transaction {
+                ChapterTable.select { ChapterTable.manga eq mangaId }
+                    .map { ChapterTable.toDataClass(it) }
+                    .toSet()
+            }
 
-        // Recognize number for new chapters.
-        chapterList.forEach { chapter ->
-            (source as? HttpSource)?.prepareNewChapter(chapter, sManga)
-            val chapterNumber = ChapterRecognition.parseChapterNumber(manga.title, chapter.name, chapter.chapter_number.toDouble())
-            chapter.chapter_number = chapterNumber.toFloat()
-        }
```

Comment:
This needs to be re-added

Reply:
Of course, I though I didn't remove it, I'll revert this